apps/docs/src/examples/inputs/TagsSuggestions.tsx

```diff
@@ -0,0 +1,114 @@
+import { useEffect, useRef, useState } from "react";
+import { Popper } from "@mui/material";
+import {
+  HvLabel,
+  HvPanel,
+  HvTag,
+  HvTagsInput,
+  HvTypography,
+} from "@hitachivantara/uikit-react-core";
+import { DropDownXS } from "@hitachivantara/uikit-react-icons";
+
+const colors = [
+  "Blue",
+  "Red",
+  "Green",
+  "Yellow",
+  "Purple",
+  "White",
+  "Black",
+  "Orange",
+  "Pink",
+  "Brown",
+  "Gray",
+  "Cyan",
+  "Magenta",
+  "Lime",
+  "Teal",
+  "Lavender",
+];
+
+const lastUsed = ["Blue", "Red", "Green"];
+
+export default function Demo() {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const [selectedColors, setSelectedColors] = useState<string[]>([]);
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.scrollLeft = containerRef.current.scrollWidth;
+    }
+  }, [selectedColors]);
+
+  const handleAddColor = (color: string) => {
+    setSelectedColors((prev) => {
+      const newColors = prev.filter((c) => c !== color);
+      return newColors.length ? [...newColors, color] : [color];
+    });
+  };
+
+  const handleRemoveColor = (color: string) => {
+    setSelectedColors((prev) => prev.filter((c) => c !== color));
+  };
+
+  return (
+    <div className="w-300px">
+      <HvLabel>Tags with suggestions</HvLabel>
+      <HvTagsInput
```

Comment:
same as above, but `HvTagsInput` takes in a `label`

```suggestion
      <HvTagsInput
        label="Tags with suggestions"
```